### DISCUSSION TITLE
Where to get the exaustive list of ISIN?

### DISCUSSION THREAD

**Discussion by kusumdev** [codereverser/casparser-isin#15] [General]
Also is there way to get in-depth details of a particular MF Fund like AUM, Historic NAV, Returns, Fund Manager, Rating, Risk etc...

**Comment by codereverser** [comment]
You could use https://nsdl.co.in/master_search_res.php .. It is also available at https://github.com/captn3m0/india-isin-data as a csv file. 

Historical NAV can be fetched from https://www.amfiindia.com/net-asset-value/nav-history. Regarding other data, I'm not sure. May be, screen scraping from other aggregators like valueresearchonline, morningstar or rupeevest, could work.

**Comment by captn3m0** [comment]
I'm now publishing Historical NAV (since 2016) as a SQLite Dataset as well: https://github.com/captn3m0/historical-mf-data (This is still in Alpha, stable release coming soon - please open issues there for any feedback).

You can also get most fund details via my MF API: https://mf.captnemo.in/ (This redirects to Kuvera API for the details you need), but it should work.
